doc/tutorials/dune-package-management/concepts.md

```diff
@@ -0,0 +1,53 @@
+# Concepts
+
+## Motivation
+
+As the OCaml ecosystem is growing, installing dependencies has become more and
+more common. As such the need for package management arose and around 2012
+[opam](https://opam.ocaml.org/) was created. `opam` enabled the creation of a
+central repository of packages called `opam-repository` which contains all
+published OCaml packages. This repository has over the years grown to about
+5000 distinct packages today.
+
+However installing dependencies with `opam` is bothersome, as `opam` always
+operates on OCaml installations (called a "switch") into which packages are
+installed and removed from. Thus when working with for multiple projects with
+different dependencies multiple installations need to be managed manually.
+
+## Improvements
+
+Dune changes the imperative process of managing an OCaml installation (switch),
+installing, updating and uninstalling packages with a declarative process where
+all the dependencies are declared in one location and then kept in sync
+automatically.
+
+The advantages of this process are as follows:
+
+  - No need to manually update the opam-repository
```

Comment:
Looking at that list without giving more context, it's unclear why any of these items are negative. 

So you can probably turn that list into a more positive feature list -- ie. focus on the positive things that `dune pkg` allows:

- automatic update of opam-repository by default, but can still control the exact opam-repo version by specifying the commit hash in `dune-project` for extra reproducibility
- package (and compilers!) are now cached - giving the illusion of having one local switch per project, but with maximal sharing (like a global switch on steroids)
- edit a file to record new dependencies (instead of using the CLI) -- this is super helpful when mixed with dune's watch mode to automatically install packages that the projects need.
- only one file to edit (`dune-project`) which becomes the source of truth -- all the rest: the opam files, the build environment, etc. is automatically kept in sync

(ie. I've tried to explain why each of your item is now a better UX with `dune pkg` instead of listing a list of opam features without context)

Reply:
I think they are negative because these are all the things that make me go "aww, finally, that's gonna be so nice".

The section is removed now because it's not really the right thing for a tutorial. When we add a reference on dune package management we should definitely write it in the way you propose.